lib/matplotlib/patches.py

```diff
@@ -27,36 +27,6 @@
 from matplotlib.bezier import make_path_regular, concatenate_paths
 
 
-# these are not available for the object inspector until after the
-# class is built so we define an initial set here for the init
-# function and they will be overridden after object definition
-docstring.interpd.update(Patch="""
```

Comment:
All looks good to me, but I'm not sure why these have been deleted?

Reply:
Because the proper version (with all the properties extracted from the docstrings) is defined at https://github.com/anntzer/matplotlib/blob/8cc0c57bc7dd638a503221f600f933efa671a046/lib/matplotlib/patches.py#L560, overwriting the old (& thus unused) definition.